### ISSUE TITLE
New Feature: Paypal - DonatePaymentForm

### ISSUE THREAD

**Issue by lauraehiller** [spacelabdev/spacelab-react#383]
NOTE: This ticket only covers building the UI and form.  Don't worry about sending data into PaypalScriptProvider, that will be another ticket.

- Can be completed after #380
- Here's what the donation form UI looks like:
![image](https://github.com/spacelabdev/spacelab-react/assets/92811107/1d308024-b0cf-4555-836a-955a1d34fa34)
We'll be updating the PaypalDonate component to look like this, but with a few changes:
- Currently in the PaypalDonate component, the user presses a button to choose whether they want to subscribe or do a one-time donation.  We should replace that functionality with the toggle feature in the design above.  
- The buttons at the bottom that are labeled "Donate with Paypal" and "Donate with Debit/Credit card" will be replaced with the <PayPalScriptProvider /> component which is inside of PaypalDonate.
- The custom amount input should only show for one-time donations
- Can remove "Add $0.00 USD to help cover the fees" and the "Write note" input
- Keep the rocket ship and moon cartoon if possible, but if it looks wonky then you can leave that piece out.  The UX team will be redesigning this page anyways so there's no need to do extra work on that.
